tests/phpunit/tests/block-bindings/render.php

```diff
@@ -198,4 +198,40 @@ public function test_update_block_with_value_from_source_image_placeholder() {
 			'The block content should be updated with the value returned by the source.'
 		);
 	}
+
+	/**
+	 * Tests if the block content is sanitized when unsafe HTML is passed.
+	 *
+	 * @ticket 60651
+	 *
+	 * @covers ::register_block_bindings_source
+	 */
+	public function test_source_value_with_unsafe_html_is_sanitized() {
```

Comment:
I'm wondering what if we should handle that case in the bindings because passing `javascript:alert('1')` to the href it is actually valid HTML. We would only be consuming the value stored in the custom field and exposed in the REST API. Not sure if it should be checked in the block bindings API, as part of the meta fields registration, or as part of the button block.